modules/java8/src/main/scala/io/circe/java8/time/JavaTimeDecoders.scala

```diff
@@ -124,6 +127,15 @@ private[time] trait JavaTimeDecoders {
         LocalDateTime.parse(input, formatter)
     }
 
+  /**
+   * @group Time
+   */
+  final def decodeMonthDayTimeWithFormatter(formatter: DateTimeFormatter): Decoder[MonthDay] =
```

Comment:
Sorry about the nitpick, but do you mind changing this to `decodeMonthDayWithFormatter` for consistency? (I can also do it in a follow-up.)

Reply:
Nice spot! I will fix it now.